File: statistics_events_callback_sample/src/main/java/com/zoho/deskportalsdk/demo/stats/events/MainActivity.java
package com.zoho.deskportalsdk.demo.stats.events;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.zoho.desk.asap.ZDPortalHome;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void launchSDK(View view) {
        ZDPortalHome.show(this);
    }
}

File: basic_sample_kotlin/src/main/java/com/zoho/deskportalsdk/demo/basic/kotlin/MainActivity.kt
package com.zoho.deskportalsdk.demo.basic.kotlin

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.zoho.desk.asap.ZDPortalHome

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun launchSDK(view: View) {
        ZDPortalHome.show(this)
    }
}
